README.md

```diff
@@ -1,23 +1,28 @@
 # Yummy_recipes
 Yummy recipes provides a platform for users to keep track of their awesome recipes and share with others if they so wish.
 
-Features:
+#Features:
```

Comment:
Add a space after the `#` to enable markdown styling. Hint: Increasing the number of `#` will reduce your heading sizes e.g `# Features`  is larger than `## Features`